static/js/compose.js

```diff
@@ -87,6 +87,10 @@ export function clear_preview_area() {
     $("#compose .preview_content").empty();
     $("#compose .markdown_preview").show();
     autosize.update($("#compose-textarea"));
+
+    // While in preview mode we hide unneeded compose_control_buttons,
+    // so here we are re-showing them.
+    hide_or_show_compose_control_buttons("show");
```

Comment:
Let's name this `toggle_compose_control_buttons`.

Reply:
Hmm, I don't think we need this function at all. Just do the hide / show in click handler itself.